src/System.Collections/src/System/Collections/Generic/SortedSet.cs

```diff
@@ -145,7 +145,7 @@ public SortedSet(IEnumerable<T> collection, IComparer<T> comparer)
                     int index = 1;
                     for (int i = 1; i < count; i++)
                     {
-                        if (comparer.Compare(els[i], els[i - 1]) != 0)
+                        if (_comparer.Compare(els[i], els[i - 1]) != 0)
```

Comment:
I was thinking of the comment being more to answer "why assign _comparer to comparer when _comparer comes from comparer?".